prov/util/src/util_fabric.c

```diff
@@ -61,7 +61,7 @@ int ofi_fabric_init(const struct fi_provider *prov,
 	atomic_initialize(&fabric->ref, 0);
 	dlist_init(&fabric->domain_list);
 	fastlock_init(&fabric->lock);
-	fabric->name = prov_attr->name;
+	fabric->name = user_attr->name;
```

Comment:
I need to examine the context around this call, but is this a safe assignment to make?  We're setting a pointer to a pointer provided by the app?